plugins/microsoft_defender_incidents/plugin.spec.yaml

```diff
@@ -0,0 +1,625 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: microsoft_defender_incidents
+title: Microsoft Defender Incidents
+description: Manage security incidents with Microsoft Defender 365
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+supported_versions: ["2022-05-06"]
+status: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/plugins/microsoft_defender_incidents
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.microsoft.com/
+tags:
+  - cloud
+  - microsoft
+  - defender
+hub_tags:
+  use_cases: [data_utility]
+  keywords: [microsoft, defender]
+  features: []
+enable_cache: false
+connection:
+  client_id:
+    title: Client ID
+    description: The application ID that the application registration portal assigned
+      to your app
+    type: string
+    required: true
+    example: 5cdad72f-c848-4df0-8aaa-ffe033e75d57
+  client_secret:
+    title: Client Secret
+    description: The application secret that you generated for your app in the app
+      registration portal
+    type: credential_secret_key
+    required: true
+    example: 5cdad72f-c848-4df0-8aaa-ffe033e75d57
+  tenant_id:
+    title: Tenant ID
+    description: This is Active Directory ID
+    type: string
+    required: true
+    example: 5cdad72f-c848-4df0-8aaa-ffe033e75d57
+types:
+  entity:
+    entityType:
+      title: Entity Type
+      description: Entities that have been identified to be part of, or related to a given alert, where possible values are - User, Ip, Url, File, Process, MailBox, MailMessage, MailCluster, and Registry
```

Comment:
Generally is is good practice to keep `e.g. type A, type B, etc` out of the description field.
If there is a limited number of choices here, would implementing these types as an `enum` be an option here?

Otherwise, this will be fine for now.

Reply:
Well when I added these options as an enum after regenerating help.md file there is no information about that choices. It just shorten description of an output fields without adding enum values anywhere. Example fields works only on inputs - I mean it puts examples just for input fields. I'm not strongly in favor of putting that output choices to the description field, but I don't see any other option right now to make it readable.

Reply:
Okay this is fine for now then :)